tests/pester-tests/commands/choco-install.Tests.ps1

```diff
@@ -2070,6 +2073,203 @@ To install a local, or remote file, you may use:
         }
     }
 
+    Context 'Installing a package with argument (<Argument>) should (<AllowsDowngrade>) downgrade an existing package dependency.' -Tag Downgrade -ForEach @(
+        @{
+            Argument        = '--force'
+            AllowsDowngrade = $true
+            ExpectedExit    = 0
+        }
+        @{
+            Argument        = '--allow-downgrade'
+            AllowsDowngrade = $true
+            ExpectedExit    = 0
+        }
+        @{
+            Argument        = ''
+            AllowsDowngrade = $false
+            ExpectedExit    = 1
+        }
+    ) {
+        BeforeAll {
+            $DependentPackage = @{
+                Name    = 'isdependency'
+                Version = '2.1.0'
+            }
+
+            Restore-ChocolateyInstallSnapshot
+
+            $Setup = Invoke-Choco install $DependentPackage.Name --version $DependentPackage.Version --confirm
+            $Output = Invoke-Choco install downgradesdependency --confirm $Argument
+            $Packages = Get-ChocolateyInstalledPackages
+        }
+
+        It "Exits correctly (<ExpectedExit>)" {
+            $Output.ExitCode | Should -Be $ExpectedExit -Because $Output.String
+        }
+
+        It "Reports that it can (<AllowsDowngrade>) downgrade isdependency" {
+            if ($AllowsDowngrade) {
+                $Output.Lines | Should -Contain 'Chocolatey installed 2/2 packages.' -Because $Output.String
+            }
+            else {
+                $Output.Lines | Should -Contain 'Chocolatey installed 0/2 packages. 2 packages failed.' -Because $Output.String
+            }
+
+            $Output.Lines | Should -Contain -Not:($AllowsDowngrade) "A newer version of $($DependentPackage.Name) (v$($DependentPackage.Version)) is already installed." -Because $Output.String
+            $Output.Lines | Should -Contain -Not:($AllowsDowngrade) 'Use --allow-downgrade or --force to attempt to install older versions.' -Because $Output.String
+        }
+
+        It "Should have the expected packages" {
+            if ($AllowsDowngrade) {
+                $Packages | where { $_.Name -eq $DependentPackage.Name -and $_.Version -eq '1.0.0' } | Should -Not -BeNullOrEmpty -Because "Packages: $Packages $($Output.String)"
+            }
+            else {
+                $Packages | where { $_.Name -eq $DependentPackage.Name -and $_.Version -eq $DependentPackage.Version } | Should -Not -BeNullOrEmpty -Because "Packages: $Packages $($Output.String)"
+            }
+        }
+    }
+
+    Context 'Installing a package with argument (<Argument>) should (<AllowsDowngrade>) downgrade an existing package dependency.' -Tag Downgrade, StopOnFirstPackageFailure -ForEach @(

```

Comment:
Should this test name mention that it includes usage of the StopOnFirstPackageFailure feature?

Reply:
It definitely should. I copied it from the other tests, and didn't update the name of it apparently.